ui/src/components/scoreboard/Scoreboard.tsx

```diff
@@ -108,25 +106,18 @@ const Scoreboard = () => {
 
     return (
         <>
-                  <Select
-                  className="border border-secondary rounded-md w-auto self-end"
-                  placeholder="Select an office"
-                  defaultValue={{ value: "All Offices", label: "All Offices" }}
-                  options={formatOptions(offices)}
-                  classNames={{
-                    menuPortal: (state) => "z-50",
-                  }}
-                  menuPortalTarget={document.body}
-                  onChange={(selected) => {
-                    setSelectedOffice(selected?.value ? selected.value : "All Offices");
-                  }}
-                />
+            <select className="border border-2 border-secondary p-1 px-3 rounded-md self-end mr-1 mb-2" placeholder="Select an office" onChange={(selected) => {
+                setSelectedOffice(selected?.target.value ? selected.target.value : "All Offices"); console.log(selectedOffice);
```

Comment:
could you remove this sneaky console.log

Reply:
no leave it, i love this kinda shit :D